cdk/l3-cdk-resources/atlas-private-endpoint/README.md

```diff
@@ -0,0 +1,166 @@
+  --execution-role-arn ROLE-ARN
+
+aws cloudformation activate-type \
+  --public-type-arn arn:aws:cloudformation:us-east-1::type/resource/bb989456c78c398a858fef18f2ca1bfc1fbba082/MongoDB-Atlas-ProjectIpAccessList \
+  --execution-role-arn ROLE-ARN
+
+aws cloudformation activate-type \
+  --public-type-arn arn:aws:cloudformation:us-east-1::type/resource/bb989456c78c398a858fef18f2ca1bfc1fbba082/MongoDB-Atlas-PrivateEndpoint \
+  --execution-role-arn ROLE-ARN  
+  
+    
+```
+
+### Minimal configuration to use this construct
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import { AtlasBasicProps } from "@mongodbatlas-awscdk/atlas-basic";
+import {AtlasBasicPrivateEndpoint, PrivateEndpointProps} from "./index";
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'atlas-basic-default', {
+    env: { region: process.env.CDK_DEFAULT_REGION, account: process.env.CDK_DEFAULT_ACCOUNT },
+});
+
+const apiKeys = {
+    privateKey: stack.node.tryGetContext('MONGODB_ATLAS_PRIVATE_KEY') || process.env.MONGODB_ATLAS_PRIVATE_KEY,
+    publicKey: stack.node.tryGetContext('MONGODB_ATLAS_PUBLIC_KEY') || process.env.MONGODB_ATLAS_PUBLIC_KEY,
+};
+
+const orgId = stack.node.tryGetContext('MONGODB_ATLAS_ORG_ID') || process.env.MONGODB_ATLAS_ORG_ID;
+const vpcId = stack.node.tryGetContext('AWS_VPC_ID') || process.env.AWS_VPC_ID;
+const subnetId = stack.node.tryGetContext('AWS_SUBNET_ID') || process.env.AWS_SUBNET_ID;
+const awsRegion = stack.node.tryGetContext('AWS_REGION') || process.env.AWS_REGION;
+
+const replicationSpecs = [
+    {
+        numShards: 1,
+        advancedRegionConfigs: [
+            {
+                analyticsSpecs: {
+                    ebsVolumeType: 'STANDARD',
+                    instanceSize: 'M10',
+                    nodeCount: 1,
+                },
+                electableSpecs: {
+                    ebsVolumeType: 'STANDARD',
+                    instanceSize: 'M10',
+                    nodeCount: 3,
+                },
+                priority: 7,
+                regionName: 'US_EAST_1',
+            },
+        ],
+    },
+];
+
+const atlasBasicProps : AtlasBasicProps = {
+    apiKeys: apiKeys,
+    clusterProps: {
+        replicationSpecs: replicationSpecs,
+    },
+    projectProps: {
+        orgId: orgId,
+    },
+}
+
+const privateEndpointProps : PrivateEndpointProps = {
+    privateEndpoints: [{
+        vpcId: vpcId,
+        subnetIds: [subnetId]
+    }],
+}
+
+const props   = {
+    apiKeys: apiKeys,
+    atlasBasicProps: atlasBasicProps,
+    privateEndpointProps: privateEndpointProps,
+    region: awsRegion
+}
+
+new AtlasBasicPrivateEndpoint(stack,'private-endpoint', props)
+```
+
+Default Region is set to us-east-1
+
+You can find more information about activating this type in the [AWS CloudFormation documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry-public.html).
```

Comment:
@SuperMohit can you add in additional section for L1 defaults in library assumed like you did here: https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/INTMDB-548/cdk/l3-cdk-resources/atlas-basic/README.md

Reply:
Also to make IP Access List open to who world we to both CIDRs: ipAddress: '0.0.0.0/1' and '128.0.0.0/1'

Reply:
I don't even want it to open for the first half. Don't think it is the right approach for the default. Default should be `deny all`

Reply:
makes sense, if we go with `deny all` then make sure this is clearly called out in your README file